mesheryctl/pkg/utils/error.go

```diff
@@ -38,6 +38,7 @@ var (
 	ErrRequestResponseCode    = "mesheryctl-1114"
 	ErrMarshalStructToCSVCode = "mesheryctl-1115"
 	ErrAppendToSheetCode      = "mesheryctl-1116"
+	ErrClearLineCode          = "mesheryctl-1117"
```

Comment:
Please do not add the error code manually, when you define a new error code leave it empty, the error code utility will auto-assign an error code.
https://docs.meshery.io/reference/error-codes

Reply:
@MUzairS15 I noticed that mesheryctl error are skipped.  I looked through the CI build code for errors and saw this

`~/errorutil update -d . -i ./server/helpers -o ./server/helpers --skip-dirs mesheryctl` This line appears to be seems to take care of updating Error codes. I noticed the error code wasn't updated when i pushed an empty new error code

Reply:
@Aijeyomah @RipulHandoo @alphaX86 do we need to change our Makefile, update our contrib docs, or update our GitHub workflow, or something else?